app/controllers/jobs_controller.rb

```diff
@@ -20,8 +20,8 @@ def create
   end
 
   def edit
-    unless @job.created_by == current_user.id
-      redirect_to root_path
+    unless @job.user_id == current_user.id
```

Comment:
So after some research, something you might want to do is have a `before_action` that you can reuse on multiple steps: 

```
  before_action :confirm_job_owner_is_current_user, only: [:edit, :destroy]

# some other code 

def confirm_job_owner_is_current_user
  redirect_to root_path unless @job.user_id == current_user.id
end
```

This also means a user can't delete a job they don't own (which is also needed)

Reply:
ahhh this is nice i like this. will add. i should consider before_actions more